src/ws.nim

```diff
@@ -164,45 +180,42 @@ proc prepareCloseBody(code: Status, reason: string): seq[byte] =
 
 proc handshake*(
   ws: WebSocket,
-  header: HttpRequestHeader,
+  request: HttpRequestRef,
   version = WSDefaultVersion) {.async.} =
   ## Handles the websocket handshake.
   ##
-
-  discard parseSaturatedNatural(header["Sec-WebSocket-Version"], ws.version)
+  let reqHeaders = request.headers
+  discard parseSaturatedNatural(reqHeaders.getString("Sec-WebSocket-Version"), ws.version)
```

Comment:
Please use `stew/base10` for parsing integers. `parseSaturatedNatural` allow `_`, ` ` and `-`.